examples/plug-and-play/demo_DPIR_mri.py

```diff
@@ -0,0 +1,158 @@
+"""
+DPIR method for PnP MRI reconstruction
+====================================================================================================
+
+This example shows how to extend the DPIR method to solve a magnetic resonance image reconstruction problem.
+The DPIR method was initially proposed in the paper:
+Zhang, K., Zuo, W., Gu, S., & Zhang, L. (2017).
+Learning deep CNN denoiser prior for image restoration.
+In Proceedings of the IEEE conference on computer vision and pattern recognition (pp. 3929-3938).
+"""
+import numpy as np
+import torch
+import deepinv as dinv
+from deepinv.physics.mri import MRIMixin
+from deepinv.optim.data_fidelity import L2
+from deepinv.optim import optim_builder, PnP
+
+
+# %%
+# Load kspace data
+# ----------------------------------------------------------------------------------------
+# In this example, we use a kspace data from the fastMRI dataset.
+
+# Set the global random seed from pytorch to ensure reproducibility of the example.
+torch.manual_seed(0)
+
+device = dinv.utils.get_freer_gpu() if torch.cuda.is_available() else "cpu"
+
+# Load kspace data
+kspace = torch.load( "tmp/kspace.pt", weights_only=True).to(device)
```

Comment:
I have just added a `deepinv.physics.MRIMixin.kspace_to_im` convenience method.
This should be directly compatible with data of shape `(B,2,...,H,W)`. 

Regardless, you should be able to just directly load FastMRI data (raw data with `deepinv.datasets.FastMRISliceDataset` and rss images with `deepinv.datasets.SimpleFastMRISliceDataset`